### PR TITLE
Challenge creation notifications

### PR DESCRIPTION
**Pull request by amickan**
- Adds follows for staff users to all site objects
- Adds notifications for staff users about newly created challenges

Closes #1936

### CODE REVIEW

File: app/tests/challenges_tests/test_models.py [hunk 1]
```diff
@@ -85,3 +88,18 @@ def test_non_posters_notified(group):
 
     assert u.user_profile.has_unread_notifications is True
     assert p.user_profile.has_unread_notifications is False
+
+
+@pytest.mark.django_db
+def test_staff_users_notified_of_challenge_creation():
+    staff_user = UserFactory(is_staff=True)
+    call_command("create_staff_follows")
```
Comment (jmsmkn) [inline app/tests/challenges_tests/test_models.py:96]: I don't think that we can run a command every time a staff user is added. I really think that this is a case where an email is fine, maybe even better would be sending it to MANAGERS (https://docs.djangoproject.com/en/3.2/ref/settings/#managers).

If you want to go the notifications route could you keep the follows in sync this via signals instead? Also, what if a staff member is removed (both deleted or if they loose privileges)?

Comment (amickan) [reply app/tests/challenges_tests/test_models.py:96]: Right, I agree that an email is probably best after all. It also leave the notification template cleaner.

Comment (amickan) [reply app/tests/challenges_tests/test_models.py:96]: As for who to send the emails to: I assume you are the only manager currently, so then you would be the only one to receive those emails - is that ok with you?

Comment (jmsmkn) [reply app/tests/challenges_tests/test_models.py]: I think that's fine! We use a mailing list for this.

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**